checker/src/org/checkerframework/checker/index/samelen/SameLenAnnotatedTypeFactory.java

```diff
@@ -73,6 +77,52 @@ public QualifierHierarchy createQualifierHierarchy(MultiGraphFactory factory) {
         return new SameLenQualifierHierarchy(factory);
     }
 
+    @Override
+    public AnnotatedTypeMirror getAnnotatedTypeLhs(Tree tree) {
+        AnnotatedTypeMirror atm = super.getAnnotatedTypeLhs(tree);
+        if (tree.getKind() == Tree.Kind.VARIABLE) {
+            String varName;
```

Comment:
Ok, you can't because org.checkerframework.framework.util.FlowExpressionParseUtil.FlowExpressionContext uses a Context which is in the framework package.